dex/elrond_dex_pair/src/lib.rs

```diff
@@ -317,6 +317,56 @@ pub trait Pair:
             .into())
     }
 
+    #[payable("*")]
+    #[endpoint(removeLiquidityAndBuyBackAndBurnToken)]
+    fn remove_liquidity_and_burn_token(
```

Comment:
Would be an option to enforce that only the farms can call this endpoint? Or a whitelisted address? I'm thinking what would happen if someone, by any chance, would call this method

Reply:
Not a fan of making the setup even harder. The function has a pretty intuitive name